components/database/composer.json

```diff
@@ -3,8 +3,8 @@
         "slim/slim": "2.*",
         "zeuxisoo/slim-whoops": "~0.2.0",
         "filp/whoops": "1.1.2",
-        "illuminate/database": "~5.5.0",
-        "illuminate/events": "~5.5.0"
+        "illuminate/database": "~6.0",
```

Comment:
Trying to decide whether we make these 6.0 or 6.0.0.. now that it's semver, TECHNICALLY it should be ok as 6.0. Gonna ask around a bit though.

Reply:
I was questioning myself exactly the same thing. In terms of composer, `~6.0` should be appropriate.

https://getcomposer.org/doc/articles/versions.md#tilde-version-range-